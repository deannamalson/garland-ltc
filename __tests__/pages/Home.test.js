import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import Home from "../../pages/index";

test("should render Home correctly", () => {
  const wrapper = shallow(<Home />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
