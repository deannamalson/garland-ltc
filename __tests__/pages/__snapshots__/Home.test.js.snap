// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home correctly 1`] = `
<div>
  <Head>
    <title>
      Home
    </title>
    <link
      href="/favicon.ico"
      rel="icon"
    />
  </Head>
  <Header />
  <Hero />
  <OurMission />
  <LicenseToCarry />
  <Contact />
  <Footer />
</div>
`;
